CollectNewspaperKarel.java

```diff
@@ -13,5 +13,25 @@
 public class CollectNewspaperKarel extends SuperKarel {
 
 	// You fill in this part
+	// Added a comment
```

Comment:
Go and read first problem statement again... Do you think that program is written the way it is said to be?